MakeMsix.targets

```diff
@@ -0,0 +1,78 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!-- Define the property MakeMsixVerbosity=high for additional logging (default=normal) -->
+  <PropertyGroup>
+    <MakeMsixVerbosity Condition="'$(MakeMsixVerbosity)' == ''">normal</MakeMsixVerbosity>
+    <MakeMsixSigned Condition="'$(MakeMsixSigned)' == ''">true</MakeMsixSigned>
+  </PropertyGroup>
+
+  <Target Name="MakeMSIX"
+    Inputs="@(MakeMsixInputs);@(MakeMsixInputsWithLocations);@(MakeMsixInputAssets);$(MakeMsixOutputFilename)"
+    Outputs="$(OutDir)$(TargetName)"
```

Comment:
How is $(MakeMsixOutputFilename) different than $(OutDir)$(TargetName)?

Reply:
Human error?